api/managementpb/ia/rules.proto

```diff
@@ -62,30 +62,42 @@ message RuleParam {
 
 // Rule represents Alert Rule.
 message Rule {
-  // Template used to create that rule.
-  Template template = 1;
   // Rule ID.
-  string rule_id = 2;
+  string rule_id = 1;
   // True if that rule is disabled.
-  bool disabled = 3;
+  bool disabled = 2;
   // Rule human-readable summary.
-  string summary = 4;
-  // Rule parameters.
-  repeated RuleParam params = 5;
-  // Rule duration.
-  google.protobuf.Duration for = 6;
-  // Rule severity.
-  management.Severity severity = 7;
+  string summary = 3;
+  // Comment.
+  string comment = 4;
+  // Expression template.
+  string expr_template = 5;
```

Comment:
Expression as it was specified in template, example: `100 / ((node_memory_MemTotal_bytes)\n/ on(node_name) group_left(service_name) (mongodb_ss_connections{conn_type=\"current\"} * 1024 * 1024))\n> [[ .threshold ]]`